lib/ui/common/recommendation_card/mock_recommendation_card_view_model.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:farmsmart_flutter/ui/common/MockString.dart';
+import 'package:farmsmart_flutter/ui/common/recommendation_card/recommendation_card.dart';
+import 'package:flutter/material.dart';
+
+class MockRecommendationCardViewModel {
+  static RecommendationCardViewModel buildRandomAddToPlotState() {
+    return RecommendationCardViewModel(
+      title: _mockTitle.random(),
+      subtitle: _mockSubtitle.random(),
+      description: _mockLargeStrings.random(),
+      leftActionText: 'View Details',
+      leftAction: () {},
+      rightActionText: 'Add to Plot',
+      rightAction: () {},
+      image: NetworkImage(_mockImageUrl.random()),
+      overlayIcon: 'assets/icons/tick_large.png',
+    );
+  }
+
+  static RecommendationCardViewModel buildRandomAddedToPlotState() {
+    return RecommendationCardViewModel(
+      title: _mockTitle.random(),
+      subtitle: _mockSubtitle.random(),
+      description: _mockLargeStrings.random(),
+      leftActionText: 'View Details',
+      leftAction: () {},
+      rightActionText: 'Added',
+      rightAction: () {},
+      image: NetworkImage('http://www.freemagebank.com/wp-content/uploads/edd/2015/07/GL0000302LR.jpg'),
+      isAdded: true,
+      overlayIcon: 'assets/icons/tick_large.png',
+    );
+  }
+
+  static RecommendationCardViewModel buildHardCodedAddToPlotState() {
+    return RecommendationCardViewModel(
+      title: 'Tomatoes',
+      subtitle: '92% match',
+      description:
+          'Tomatoes are lorem ipsum dolor sit amet consectetur elit sed'
+          ' do eiusmod tempor lorem ipsum dolor sit amet',
+      leftActionText: 'View Details',
+      leftAction: () {},
+      rightActionText: 'Add to Plot',
+      rightAction: () {},
+      image: NetworkImage(_mockImageUrl.random()),
```

Comment:
This might need to be a future, as the image URLS come from a async source. but we can deal with that when we integrate.

Reply:
👍